lib/index.js

```diff
@@ -343,3 +372,13 @@ function _updateSessionAuthenticationData(
   }
   return data;
 }
+
+function _toBase64NoPad(x) {
+  return x.toString('base64', 0, 16).split('=')[0];
```

Comment:
Can use limit since only using first result.
```suggestion
  return x.toString('base64', 0, 16).split('=', 1)[0];
```

Reply:
@dlongley mentioned this [also](https://github.com/digitalbazaar/bedrock-authn-token-http/pull/31#discussion_r1223152662), but this can be simplified further by passing `base64Url` as the encoding in `toString`

```js
return x.toString('base64Url', 0, 16);
```

Reply:
Well, it turns out that bedrock-authn-token is using just `base64` so I think we have to keep what we have here, `base64url` uses the wrong alphabet.